src/supervised/scaledloss.jl

```diff
@@ -1,4 +1,4 @@
-for KIND in (:MarginLoss, :DistanceLoss)
+for KIND in (:MarginLoss, :DistanceLoss, :SupervisedLoss)
```

Comment:
a few lines down there is a `typealias` for `ScaledLoss` which should reflect this addition